src/Microsoft.Sbom.Api/Utils/ComponentDetectionCliArgumentBuilder.cs

```diff
@@ -203,5 +203,15 @@ public ComponentDetectionCliArgumentBuilder ParseAndAddArgs(string args)
 
             return this;
         }
+
+        private string AsArgumentValue(string arg)
```

Comment:
this implementation is scary simple. dotnet does not yet have a proper api to call external processes?